### PR TITLE
fix(menu): background and shadow colors

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Comment by rachelbt** [comment]
@jshenkman 
menu: https://tmp.dev.vivid.vonage.com/1280/index.html?path=/story/components-menu--with-vwc-list-item
dropdown: https://tmp.dev.vivid.vonage.com/1280/index.html?path=/story/alpha-components-dropdown--basic&args=corner:BOTTOM_START

**Comment by jshenkman** [comment]
> @jshenkman menu: https://tmp.dev.vivid.vonage.com/1280/index.html?path=/story/components-menu--with-vwc-list-item dropdown: https://tmp.dev.vivid.vonage.com/1280/index.html?path=/story/alpha-components-dropdown--basic&args=corner:BOTTOM_START

LGTM!! 🤩😍

File: components/surface/src/vwc-surface.scss [hunk 1]
```diff
@@ -5,8 +5,9 @@ $surface-border-radius: --vvd-surface-border-radius;
 
 :host {
 	.mdc-menu-surface {
-		background-color: var(#{scheme-variables.$vvd-color-canvas});
+		background-color: var(--vvd-color-surface-8dp);
```
Comment (yinonov) [inline components/surface/src/vwc-surface.scss:8]: Change to sass const?

File: components/surface/src/vwc-surface.scss [hunk 2]
```diff
@@ -5,8 +5,9 @@ $surface-border-radius: --vvd-surface-border-radius;
...
 		border-radius: var(#{$surface-border-radius}, $default-surface-border-radius);
-		box-shadow: 0 6px 8px rgba(0, 0, 0, 0.05), 0 12px 16px rgba(0, 0, 0, 0.05), 0 4px 24px rgba(0, 0, 0, 0.1);
+		box-shadow: none; /*to remove the mdc box-shadow*/
+		filter: var(--vvd-shadow-surface-8dp);
```
Comment (yinonov) [inline components/surface/src/vwc-surface.scss:11]: Same here

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — CHANGES_REQUESTED** [review]
should select be scoped to this fix

image

**Comment by rachelbt** [comment]
> should select be scoped to this fix
> 
> image

basically yes - the problem is that in the select we are using the `mwc-menu-surface` and not the vivid one.
If this need to be fixed - its a different task for my opinion

**Review by yinonov — APPROVED** [review]

**Review by yinonov — CHANGES_REQUESTED** [review]
checking

**Review by yinonov — APPROVED** [review]

**Pull request merged**